src/test/java/org/rocstreaming/roctoolkit/BaseTest.java

```diff
@@ -1,17 +1,35 @@
 package org.rocstreaming.roctoolkit;
 
+import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 
-import java.io.IOException;
 import java.io.InputStream;
+import java.util.logging.Level;
 import java.util.logging.LogManager;
 
 public class BaseTest {
 
+    static private Level originalLogLevel;
+
     @BeforeAll
-    static void configureLogger() throws IOException {
+    static void configureLogger() throws Exception {
         try (InputStream is = RocReceiverTest.class.getClassLoader().getResourceAsStream("logging.properties")) {
             LogManager.getLogManager().readConfiguration(is);
         }
+
+        originalLogLevel = RocLogger.LOGGER.getLevel();
+
+        String gradleLogLevel = System.getProperty("org.gradle.logging.level");
```

Comment:
maybe try this https://stackoverflow.com/a/56742778 ?
to configure this in gradle config